.gitattributes

```diff
@@ -5,12 +5,13 @@ thirdparty/* linguist-vendored
 
 # Normalize EOL for all files that Git considers text files
 * text=auto eol=lf
-# Except for bat files, which are Windows only files
+# Except for Windows-only / Visual Studio files
 *.bat eol=crlf
+*.sln eol=crlf
+*.csproj eol=crlf
```

Comment:
`*.csproj` are not Visual Studio specific, they are .NET-specific. If I run `dotnet new console` on my Mac I get a `*.csproj` file with LF line endings. The official tooling uses LF, so it is incorrect to use CRLF here.

Reply:
I've heard mixed opinions on how `*.csproj` should be handled. Seeing that it's automatically handled as `lf` on Mac makes me wonder why the ["default" C# `.gitattributes`](https://github.com/gitattributes/gitattributes/blob/6254cbbe9d8b9d7b6c719a2f57c69969a3aab476/CSharp.gitattributes) instead specifies `crlf`

Reply:
In Linux `dotnet new console` produces `*.sln` and `*.csproj` files that are UTF-8 with BOM and CRLF for me. According to https://github.com/dotnet/templating/issues/1248#issuecomment-331340408 this is by design in order to avoid issues with Windows tooling. If that's the case I think it'd be preferable to use a line ending that ensures it works in every platform.